ng-dev/pr/merge/cli.ts

```diff
@@ -28,6 +27,7 @@ async function builder(argv: Argv) {
     .strict()
     .positional('pr', {
       demandOption: true,
+      coerce: (prUrlOrNumber) => parsePrNumber(prUrlOrNumber),
       type: 'number',
```

Comment:
Evidently the type determines the parsing that is done by yargs automatically, then it sees the type as the result of the `coerce` function, so we need to declare the type as `string` here, and then it will get coerced into number.

```suggestion
      coerce: (prUrlOrNumber: string) => parsePrNumber(prUrlOrNumber),
      type: 'string',
```